src/.nuget/packages.config

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
```

Comment:
This is for Linux and I used it to specify runtime (I need different for the different systems, it seems). Still not sure how to deal with this.